CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
-* If you have found a discrepancy in documented and observed behaviour, that
```

Comment:
Sorry to put another thing on you, but I've pondered these (presumably prettier-inspired) formatting changes a bit longer,  and I've decided that I can't live with prettier trying to copy-edit my markdown.  I'll be spending the rest of my life making "make prettier happy" commits, to comply with a markdown style that I find pretty unpleasant and unnatural.  Please revert the formatting-only changes to markdown files, and teach prettier to leave them alone for the future.  Thanks.

Reply:
Sure :)

Reply:
I'll get prettier to ignore everything that isn't a JavaScript file (since that's what I originally added it for) and revert all other changes